docker-compose-config-c.yaml

```diff
@@ -69,6 +78,7 @@ services:
     volumes:
       - bitcoin_pool_side_data:/root/.bitcoin
       - ./custom-configs/sri-roles/bitcoin-tp-pool.conf:/root/.bitcoin/bitcoin.conf
+      - ./logs:/var/logs
```

Comment:
Why do you add this?

Reply:
Nice catch !! Removed with ace6ff986dead2583668a87adf88b2189e72bfd1